src/framework/chart/LineChart.jsx

```diff
@@ -82,10 +119,20 @@ export default class LineChart extends Component {
 
     const duration = updateImmediately ? 0 : transitionDuration;
     const marginBottom = 14;
-    const marginRight = yAxisLabelWidth;
+    const marginTop = 5;
     const width = this.$lineChart.width();
-    const availableWidth = width - marginRight;
-    const availableHeight = height - marginBottom;
+    const yAxisMax = height - marginBottom;
+    const yAxisTickSize = width - yAxisLabelWidth;
+
+    let marginLeft = 5;
```

Comment:
For it to be flexible as part of a generic UI toolkit, it would be a good (and reasonably small) investment to move  `marginTop`, `marginBottom` as props whereas `marginHorizontal` can be something that can be `marginLeft` or `marginRight` depending what value `YAxisLeft` has.